src/features/content-answers/ContentAnswers.tsx

```diff
@@ -17,27 +17,35 @@ type Props = {
 
 const ContentAnswers = ({ file }: Props) => {
     const [isModalOpen, setIsModalOpen] = useState(false);
+    const [isHighlighted, setIsHighlighted] = useState(false);
 
-    const handleClick = () => {
+    const handleClick = useCallback(() => {
         setIsModalOpen(true);
-    };
+    }, [setIsModalOpen]);
 
-    const handleClose = () => {
+    const handleClose = useCallback(() => {
         setIsModalOpen(false);
-    };
+    }, [setIsModalOpen]);
+
+    const handleAsk = useCallback(() => {
```

Comment:
do these need the `useCallback`? is there a benefit to them?

Reply:
the performance difference is probably incredible low from our point of view but I dont see it hurting us in the short term.